src/test/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapperTest.java

```diff
@@ -43,11 +46,13 @@ void mapsEventMapperToEventHandlerEntityCorrectly() {
             .hasFieldOrPropertyWithValue("createdBy", USER_ACCOUNT_ENTITY);
     }
 
-    @Test
-    void mapsEventMapperToEventHandlerEntityCorrectlyWhenHandlerIsEmpty() {
+    @ParameterizedTest
+    @NullSource
+    @ValueSource(strings = {""})
```

Comment:
Perhaps also test for blank string e.g.:
```suggestion
    @ValueSource(strings = {"", " "})
```